$NetBSD: patch-src_mesa_drivers_x11_Makefile,v 1.1 2015/04/05 17:01:37 tnn Exp $

Build the x11 version libGL.so with version numbering major 1 and minor 2,
the same a the glx version of libGL (simplifies PLIST handling).

--- src/mesa/drivers/x11/Makefile.orig	2011-10-15 00:43:58.000000000 +0000
+++ src/mesa/drivers/x11/Makefile
@@ -61,7 +61,7 @@ default: $(TOP)/$(LIB_DIR)/$(GL_LIB_NAME
 
 $(TOP)/$(LIB_DIR)/$(GL_LIB_NAME): $(OBJECTS) $(CORE_MESA)
 	@ $(MKLIB) -o $(GL_LIB) -linker '$(CXX)' -ldflags '$(LDFLAGS)' \
-		-major $(GL_MAJOR) -minor $(GL_MINOR) -patch $(GL_TINY) \
+		-major 1 -minor 2 \
 		-install $(TOP)/$(LIB_DIR) \
 		-cplusplus $(MKLIB_OPTIONS) \
 		-id $(INSTALL_LIB_DIR)/lib$(GL_LIB).$(GL_MAJOR).dylib \
